src/components/ConnectWallet/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useWeb3React } from '@web3-react/core'
+import { connections, defaultChainIdAtom, getConnectorName } from 'hooks/connectWeb3/useWeb3React'
+import { useAtomValue } from 'jotai/utils'
+import { useEffect } from 'react'
+
+import ConnectedWalletChip from './ConnectedWalletChip'
+import ConnectWallet from './ConnectWallet'
+
+interface WalletProps {
+  disabled?: boolean
+  onConnectWalletClick?: (e?: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+export default function Wallet({ disabled, onConnectWalletClick }: WalletProps) {
+  // Attempt to connect eagerly on mount, and prompt switch networks when integrator's defaultChainId changes
+  const defaultChainId = useAtomValue(defaultChainIdAtom)
+  useEffect(() => {
+    connections.forEach(([wallet, _]) =>
+      wallet.connectEagerly
+        ? wallet.connectEagerly(defaultChainId)?.catch(() => {
+            console.log('Could not connect eagerly to', getConnectorName(wallet))
+          })
+        : wallet.activate(defaultChainId)?.catch(() => {
+            console.log('Could not activate', getConnectorName(wallet))
+          })
+    )
+  }, [defaultChainId])
+
+  const { account, isActive } = useWeb3React()
+
+  const isConnected = isActive && Boolean(account)
```

Comment:
why does account need to be defined for isConnected to be true?

Reply:
will rename as `isAccountConnected` -- if we're connected to an account, then we want to display the connected wallet chip:
<img width="151" alt="Screen Shot 2022-08-02 at 12 14 48 PM" src="https://user-images.githubusercontent.com/27475332/182422925-d5c8856f-3112-458a-976b-fd21ca336633.png">

else, display this:
<img width="185" alt="image" src="https://user-images.githubusercontent.com/27475332/182423027-77576b46-d3c1-4c50-8f51-b60e525e095a.png">